### ISSUE TITLE
Putting a slashes (`/`) in filename when scanning, does not save the file nor shows an error

### ISSUE THREAD

**Issue by stavros-k** [nextcloud/ios#2880]
### Steps to reproduce

1. Open application
2. Scan a document
3. Enter a file name like `2024/04/01`
4. Hit save

### Expected behaviour

Expect to see an error that `/` are not supported in filenames

Tell us what should happen.

### Actual behaviour

Screen goes back to the folder view, file is not saved, error not shown.

Tell was what instead happens.

### Screenshots

If applicable, add a screenshot showing the issue.

### Logs

```
If applicable, you can post the iOS app or server logs (removing any sensitive information).
```

### Reasoning or why should it be changed/implemented?

I've spent **2 hours** trying to teach my mother to scan her documents. :D 
Until I realized that she was adding forward slashes and the files was not saving.

TL;DR; is not intuitive for all people that filenames should not include `/`!

### Environment data

**iOS version:** e.g. iOS 17.4.1

**Nextcloud iOS app version:** see More > Settings 

5.2.7

**Server operating system:**

28.0.4

**Web server:** Apache, nginx

**Database:**

Postgres

**PHP version:**

**Nextcloud version:** see Nextcloud admin page

**Comment by cmck-xyz** [comment]
Similar issue but stuck in error loop after trying to create a directory with a "/" in the name.

**Comment by mpivchev** [comment]
Hi,
I tested on my side and I could not reproduce. Typing `2024/04/01` does show an error.

![Image](https://github.com/user-attachments/assets/1d4213b6-4d0c-4d59-86b8-39c5f7b1d55a)
